snap/README.md

```diff
@@ -114,35 +146,322 @@ Or by using the systemd unit name and `journalctl`:
+|messagequeue.subscribe-topic | SubscribeTopic indicates the topic in which to subscribe.|
+| messagequeue.auth-mode | AuthMode specifies the type of secure connection to the message bus which are 'none', 'usernamepassword','clientcert' or 'cacert'. Not all option supported by each implementation. ZMQ doesn't support any Authmode beyond 'none', RedisStreams only supports 'none' & 'usernamepassword' while MQTT supports all options.|
+|messagequeue.secret-name|  SecretName is the name of the secret in the SecretStore that contains the Auth Credentials. The credential are dynamically loaded using this name and store the Option property below where the implementation expected to find them.|
+|messagequeue.subscribe-enabled| SubscribeEnabled indicates whether enable the subscription to the Message Queue|
+
+| [Trigger]   | Description |
+| --------------------- | ----------- |
+| trigger.edgex-message-bus.subscribe-host.port||
+| trigger.edgex-message-bus.subscribe-host.protocol||
+|trigger.edgex-message-bus.subscribe-host.subscribe-topics||
+|trigger.edgex-message-bus.publish-host.port ||
+|trigger.edgex-message-bus.publish-host.protocol||
+|trigger.edgex-message-bus.publish-host.publish-topic||
+
+
+
+### API Gateway settings (prefix: env.security-proxy.)
+
+
+
+| API Gateway setting   | Description |
+| --------------------- | ----------- |
+| add-proxy-route | The add-proxy-route setting is a csv list of URLs to be added to the API Gateway (aka Kong). See [documentation](https://docs.edgexfoundry.org/1.3/microservices/security/Ch-APIGateway/) NOTE - this setting is not a configuration override, it's a top-level environment variable used by the security-proxy-setup. |
+
+
+### Secret Store settings (prefix: env.security-secret-store.)
+
+| API Gateway setting   | Description |
+| --------------------- | ----------- |
+| add-secretstore-tokens | The add-secretstore-tokens setting is a csv list of service keys to be added to the list of Vault tokens that security-file-token-provider (launched by security-secretstore-setup) creates. It is set to a default list of additional services by the snap, so be sure to examine the default setting before providing a custom list of services. NOTE - this setting is not a configuration override, it's a top-level environment variable used by the security-secretstore-setup. |
+| add-known-secrets | The add-known-secrets setting is a csv list of secret paths and associated services. It's used to provision the specified secret for the given service in Vault. It is set to a default list of additional services by the snap, so be sure to examine the default setting before providing a custom list of services. NOTE - this setting is not a configuration override, it's a top-level environment variable used by the security-secretstore-setup.|
+
+
+### Security bootstrapper settings (prefix: env.security-bootstrapper.)
+
+| setting   | Description |
+| --------------------- | ----------- |
+|add-registry-acl-roles| |
```

Comment:
description needed... I think you can base this on one of the above descriptions.